apps/plopdown-ext/src/manifest.json

```diff
@@ -5,13 +5,24 @@
   "homepage_url": "https://plopdown.video",
   "description": "Show annotations on top of any html5 web video.",
   "default_locale": "en",
-  "permissions": ["activeTab", "storage", "unlimitedStorage"],
+  "permissions": ["activeTab", "storage", "unlimitedStorage", "<all_urls>"],
```

Comment:
this is one of those permissions that'll get extra scrutiny security-wise, any thoughts on how we could make it optional?